### ISSUE TITLE
Ship ArcCommander application as dotnet tool

### ISSUE THREAD

**Issue by HLWeil** [nfdi4plants/arcCommander#5]
Test, pack and upload application

**Comment by kMutagene** [comment]
This might be closed, only thing missing is actually publishing the package

**Comment by Brilator** [comment]
:D

**Comment by Brilator** [comment]
https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools-how-to-create

**Comment by kMutagene** [comment]
@Brilator i think one of the main reasons why this never happened is people strictly not wanting to install dotnet for some reason, so we invested time into bundling the application in a self-contained manner (which might also be the reason for some of the performance problems)

**Comment by Brilator** [comment]
Yes, I know. Sorry, I just talked to @HLWeil while commenting here, who pointed out that this was a direction some years back ;)

**Comment by Brilator** [comment]
All fun aside. With ARCitect now being the user-ready primary tool, it might makes sense to still publish via nuget. 
Teaching how to install .NET and a tool from there is so much easier than the hassle with PATHs.

**Comment by omaus** [comment]
> [@Brilator](https://github.com/Brilator) i think one of the main reasons why this never happened is people strictly not wanting to install dotnet for some reason, so we invested time into bundling the application in a self-contained manner (which might also be the reason for some of the performance problems)

We also have the application on winget, which is easier to use and requires in, as far as I remember, Win11 cases no installation.

**Comment by kMutagene** [comment]
@omaus @HLWeil 

> We also have the application on winget

Please add the installation sources in short to the readme directly, You can perfectly see why: The current links to the manuals in the readme are broken, so there is currently no way to find out starting from this repo.

**Comment by kMutagene** [comment]
Also, if there is a dedicated website for the arccommander, i'd suggest linking it as repo website
